dozer-sql/src/pipeline/expression/builder.rs

```diff
@@ -396,6 +396,13 @@ impl ExpressionBuilder {
             return self.parse_python_udf(udf_name, sql_function, schema);
         }
 
+        #[cfg(feature = "wasm")]
```

Comment:
Would be nice if instead of having a `wasm_` prefix, wasm based UDFs are declared in the config file. Do not follow  too much the Python sample as we have to refactor it.